nix/node-env.nix

```diff
@@ -84,6 +84,16 @@ let
 
       # Change to the package directory to install dependencies
       cd "$DIR/$packageName"
+
```

Comment:
It would be nice if this can be rewritten in JavaScript code and executed by Node.js. This is what we do in other pieces of the script as well.

Otherwise, we need to introduce yet another build-time dependency.